paddle/pten/kernels/cpu/split_kernel.cc

```diff
@@ -69,4 +69,5 @@ PT_REGISTER_KERNEL(split,
                    int64_t,
                    int,
                    bool,
-                   pten::dtype::float16) {}
+                   pten::dtype::float16,
```

Comment:
The file path has changed, maybe you need to rebase the local code and submit it again.